modules/userId/index.js

```diff
@@ -743,14 +743,14 @@ function registerSignalSources() {
   if (!isGptPubadsDefined()) {
     return;
   }
-  window.googletag.encryptedSignalProviders = window.googletag.encryptedSignalProviders || [];
+  window.googletag.secureSignalProviders = window.googletag.secureSignalProviders || [];
   const encryptedSignalSources = config.getConfig('userSync.encryptedSignalSources');
   if (encryptedSignalSources) {
     const registerDelay = encryptedSignalSources.registerDelay || 0;
     setTimeout(() => {
       encryptedSignalSources['sources'] && encryptedSignalSources['sources'].forEach(({ source, encrypt, customFunc }) => {
         source.forEach((src) => {
-          window.googletag.encryptedSignalProviders.push({
+          window.googletag.secureSignalProviders.push({
             id: src,
```

Comment:
I don't know how to test this - but `id` does not appear in the [docs you linked](https://developers.google.com/publisher-tag/reference#javascript_65).

Reply:
Sorry that I linked to the wrong section in the doc. Please check this link instead:
https://developers.google.com/publisher-tag/reference#googletag.securesignals.biddersignalprovider